sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/jackson/StdConverters.java

```diff
@@ -10,11 +10,13 @@
 
 abstract class StdConverters {
 
+    private static final String FIELD_NAME_OF_VALUE = "value";
+
     static final class ClientAuthenticationMethodConverter extends StdConverter<JsonNode, ClientAuthenticationMethod> {
 
         @Override
         public ClientAuthenticationMethod convert(JsonNode jsonNode) {
-            String value = JsonNodeUtils.findStringValue(jsonNode, "value");
```

Comment:
Fix issue: Define a constant instead of duplicating this literal "value" 3 times.